### PR TITLE
Added monitoring sidecar for Heapster

### PR DESCRIPTION
**Pull request by piosz**
```release-note
Collect metrics from Heapster in Stackdriver mode.
```

### CODE REVIEW

File: cluster/addons/cluster-monitoring/stackdriver/heapster-controller.yaml [hunk 1]
```diff
@@ -64,6 +64,15 @@ spec:
             - name: usr-ca-certs
               mountPath: /usr/share/ca-certificates
               readOnly: true
+        - name: prometheus-to-sd-exporter
+          image: gcr.io/google-containers/prometheus-to-sd:v0.2.1
+          command:
+            - /monitor
+            - --source=heapster:http://localhost:8082?whitelisted=stackdriver_requests_count,stackdriver_timeseries_count
+            - --stackdriver-prefix=container.googleapis.com/internal/addons
```
Comment (loburm) [inline cluster/addons/cluster-monitoring/stackdriver/heapster-controller.yaml]: Considering that this sidecar monitors only heapster, maybe we also should pass pod_id and namespace_id here?

Comment (piosz) [inline cluster/addons/cluster-monitoring/stackdriver/heapster-controller.yaml]: done

**Comment by piosz** [comment]
@loburm I checked that it works. PTAL

**Comment by piosz** [comment]
@wojtek-t this should be cherry-picked to 1.7

**Comment by wojtek-t** [comment]
@piosz - please add a release note

**Comment by piosz** [comment]
@wojtek-t done

**Comment by wojtek-t** [comment]
Cherrypick approved.

@piosz - cherrypick can't be automatially made due to conflicts with 1.7 branch. Please prepare cherrypick yourself and I will approve it.

**Pull request merged**